File: SiriusQuality/SiriusQuality-ET/src/java/SiriusQuality_ET/EnergyBalanceCompositeAuxiliary.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class EnergyBalanceCompositeAuxiliary
{
    private double maxTair;
    private double minTair;
    private double vaporPressure;
    private double extraSolarRadiation;
    private double solarRadiation;
    private double plantHeight;
    private double wind;
    private double hslope;
    private double VPDair;
    private double netOutGoingLongWaveRadiation;
    private double netRadiation;
    private double netRadiationEquivalentEvaporation;
    
    public EnergyBalanceCompositeAuxiliary() { }
    
    public EnergyBalanceCompositeAuxiliary(EnergyBalanceCompositeAuxiliary toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.maxTair = toCopy.getmaxTair();
            this.minTair = toCopy.getminTair();
            this.vaporPressure = toCopy.getvaporPressure();
            this.extraSolarRadiation = toCopy.getextraSolarRadiation();
            this.solarRadiation = toCopy.getsolarRadiation();
            this.plantHeight = toCopy.getplantHeight();
            this.wind = toCopy.getwind();
            this.hslope = toCopy.gethslope();
            this.VPDair = toCopy.getVPDair();
            this.netOutGoingLongWaveRadiation = toCopy.getnetOutGoingLongWaveRadiation();
            this.netRadiation = toCopy.getnetRadiation();
            this.netRadiation = toCopy.getnetRadiation();
            this.netRadiationEquivalentEvaporation = toCopy.getnetRadiationEquivalentEvaporation();
        }
    }
    public double getmaxTair()
    { return maxTair; }

    public void setmaxTair(double _maxTair)
    { this.maxTair= _maxTair; } 
    
    public double getminTair()
    { return minTair; }

    public void setminTair(double _minTair)
    { this.minTair= _minTair; } 
    
    public double getvaporPressure()
    { return vaporPressure; }

    public void setvaporPressure(double _vaporPressure)
    { this.vaporPressure= _vaporPressure; } 
    
    public double getextraSolarRadiation()
    { return extraSolarRadiation; }

    public void setextraSolarRadiation(double _extraSolarRadiation)
    { this.extraSolarRadiation= _extraSolarRadiation; } 
    
    public double getsolarRadiation()
    { return solarRadiation; }

    public void setsolarRadiation(double _solarRadiation)
    { this.solarRadiation= _solarRadiation; } 
    
    public double getplantHeight()
    { return plantHeight; }

    public void setplantHeight(double _plantHeight)
    { this.plantHeight= _plantHeight; } 
    
    public double getwind()
    { return wind; }

    public void setwind(double _wind)
    { this.wind= _wind; } 
    
    public double gethslope()
    { return hslope; }

    public void sethslope(double _hslope)
    { this.hslope= _hslope; } 
    
    public double getVPDair()
    { return VPDair; }

    public void setVPDair(double _VPDair)
    { this.VPDair= _VPDair; } 
    
    public double getnetOutGoingLongWaveRadiation()
    { return netOutGoingLongWaveRadiation; }

    public void setnetOutGoingLongWaveRadiation(double _netOutGoingLongWaveRadiation)
    { this.netOutGoingLongWaveRadiation= _netOutGoingLongWaveRadiation; } 
    
    public double getnetRadiation()
    { return netRadiation; }

    public void setnetRadiation(double _netRadiation)
    { this.netRadiation= _netRadiation; } 
    
    public double getnetRadiationEquivalentEvaporation()
    { return netRadiationEquivalentEvaporation; }

    public void setnetRadiationEquivalentEvaporation(double _netRadiationEquivalentEvaporation)
    { this.netRadiationEquivalentEvaporation= _netRadiationEquivalentEvaporation; } 
    
}

File: SiriusQuality/SiriusQuality-ET/src/cpp/SiriusQuality_ET/EnergyBalanceCompositeAuxiliary.cpp
#include "EnergyBalanceCompositeAuxiliary.h"
using namespace SiriusQuality_ET;

EnergyBalanceCompositeAuxiliary::EnergyBalanceCompositeAuxiliary() {}

double EnergyBalanceCompositeAuxiliary::getmaxTair() { return this->maxTair; }
double EnergyBalanceCompositeAuxiliary::getminTair() { return this->minTair; }
double EnergyBalanceCompositeAuxiliary::getvaporPressure() { return this->vaporPressure; }
double EnergyBalanceCompositeAuxiliary::getextraSolarRadiation() { return this->extraSolarRadiation; }
double EnergyBalanceCompositeAuxiliary::getsolarRadiation() { return this->solarRadiation; }
double EnergyBalanceCompositeAuxiliary::getplantHeight() { return this->plantHeight; }
double EnergyBalanceCompositeAuxiliary::getwind() { return this->wind; }
double EnergyBalanceCompositeAuxiliary::gethslope() { return this->hslope; }
double EnergyBalanceCompositeAuxiliary::getVPDair() { return this->VPDair; }
double EnergyBalanceCompositeAuxiliary::getnetOutGoingLongWaveRadiation() { return this->netOutGoingLongWaveRadiation; }
double EnergyBalanceCompositeAuxiliary::getnetRadiation() { return this->netRadiation; }
double EnergyBalanceCompositeAuxiliary::getnetRadiationEquivalentEvaporation() { return this->netRadiationEquivalentEvaporation; }

void EnergyBalanceCompositeAuxiliary::setmaxTair(double _maxTair) { this->maxTair = _maxTair; }
void EnergyBalanceCompositeAuxiliary::setminTair(double _minTair) { this->minTair = _minTair; }
void EnergyBalanceCompositeAuxiliary::setvaporPressure(double _vaporPressure) { this->vaporPressure = _vaporPressure; }
void EnergyBalanceCompositeAuxiliary::setextraSolarRadiation(double _extraSolarRadiation) { this->extraSolarRadiation = _extraSolarRadiation; }
void EnergyBalanceCompositeAuxiliary::setsolarRadiation(double _solarRadiation) { this->solarRadiation = _solarRadiation; }
void EnergyBalanceCompositeAuxiliary::setplantHeight(double _plantHeight) { this->plantHeight = _plantHeight; }
void EnergyBalanceCompositeAuxiliary::setwind(double _wind) { this->wind = _wind; }
void EnergyBalanceCompositeAuxiliary::sethslope(double _hslope) { this->hslope = _hslope; }
void EnergyBalanceCompositeAuxiliary::setVPDair(double _VPDair) { this->VPDair = _VPDair; }
void EnergyBalanceCompositeAuxiliary::setnetOutGoingLongWaveRadiation(double _netOutGoingLongWaveRadiation) { this->netOutGoingLongWaveRadiation = _netOutGoingLongWaveRadiation; }
void EnergyBalanceCompositeAuxiliary::setnetRadiation(double _netRadiation) { this->netRadiation = _netRadiation; }
void EnergyBalanceCompositeAuxiliary::setnetRadiationEquivalentEvaporation(double _netRadiationEquivalentEvaporation) { this->netRadiationEquivalentEvaporation = _netRadiationEquivalentEvaporation; }